niwidgets/exampledata.py

```diff
@@ -25,3 +25,5 @@
     surface_dir / f
     for f in ('lh.area', 'lh.curv', 'lh.thickness', 'lh.aparc.annot')
 ]
+
+streamlines = root_dir / 'streamlines.trk'
```

Comment:
Because this returns a PosixPath object, the example in the index file raises an error, unless I pass `sw = StreamlineWidget(filename=streamlines.as_posix())`